src/components/ReviewCard/ReviewCard.jsx

```diff
@@ -23,7 +23,7 @@ const SubheaderReview = ({ createdAt, isStillWorkingHere, jobTitle }) => (
     <Box>
         <Typography variant="body2">{jobTitle}</Typography>
         <Typography variant="body2">
-            {`(${isStillWorkingHere ? 'Current Employee' : 'Former Employee'})`} - {createdAt}
+            {`(${isStillWorkingHere === 1 ? 'Current Employee' : 'Former Employee'})`} - {createdAt}

```

Comment:
The number one works as a **true value**. **Specifying that it is equal to 1 is not necessary**.